HW06.Rmd

```diff
@@ -68,7 +82,14 @@ GBM.genes.of.interest <- filter(ens.mapped_GBM, gene_symbol %in% genes_of_intere
 
 #Now perform a t-test between the H3K mutated and the wt samples. There are many ways to do this actually, you can use a for loop or you could do the tidy alternative with broom(), but the for loop is probably the easiest
 
-#print out the t-test results
+for (gene in c(1:26)) {
+  H3K27_Mutated <- as.vector(GBM.genes.of.interest[gene,(2:24)])
+  WT <- as.vector(GBM.genes.of.interest[gene,(25:46)])
+  ttest <- t.test(H3K27_Mutated,WT)
+  #Print out the t-test results
+  print(ttest)
+}
+
 ```

```

Comment:
I think this is a nice way to make the loop that I did not really think of. However, for the results, now I cannot really see which t-test is for which gene, and I think it might be useful to provide that information.

Reply:
I agree...the results of t-test are pretty hard to read. I will re-write this part.